### PR TITLE
configure apache to log to stdout/stderr

### PR DESCRIPTION
**Pull request by paulczar**
Apache is configured by default to log to `/var/log...` which fills up real quick with logs from kubernetes health checks and regular traffic, eventually the pod runs out of space.  This modifies the config to log to stdout/stderr instead inline with docker/kubernetes best practices and should help protect the filesystem of the containers/pods.

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

### CODE REVIEW

**Comment by paulczar** [comment]
This is an example change of course, as it would be needed across all of the different spinnaker microservices, not just deck.

File: docker/run-apache2.sh [hunk 1]
```diff
@@ -44,6 +44,10 @@ mv spinnaker.conf  /etc/apache2/sites-available
 
 a2ensite spinnaker
 
+# Disable default site
+
+a2dissite 000-default
```
Comment (paulczar) [inline docker/run-apache2.sh]: we're not using the default, so its good hygiene to disable it.

File: docker/setup-apache2.sh [hunk 2]
```diff
@@ -1,5 +1,7 @@
 apt-get update
 apt-get install apache2 -y
+rm -rf /var/lib/apt/lists/*
```
Comment (paulczar) [inline docker/setup-apache2.sh]: there's a bit of space to save by cleaning up the apt cache.

**Comment by ezimanyi** [comment]
@Jammy-Louie : I'm not sure I'm the best person to look at this PR...is there someone more familiar with deck you could ask?

**Comment by Jammy-Louie** [comment]
@ezimanyi Sure but this looks like a change we would want to adopt in all the other microservices and it is not just applicable to deck

**Pull request merged**